docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+* `pip`: version 8 or higher
+* If you haven't done so already, install the gRPC runtime and gRPC tools
```

Comment:
I don't think this line belongs here because how to make this two installations is the work of the next two `###`-delimited sections.